docs/source/ProgrammingGuide/Subviews.md

```diff
@@ -10,20 +10,20 @@ After reading this chapter, you should understand the following:
 ## 11.1 A subview is a slice of a View
 
 In Kokkos, a _subview_ is a slice of a View. A _slice_ of a multidimensional array behaves as an array, and is a view of a
-structured subset of the original array. "Behaves as an array" means that the slice has the same syntax as an array does; one can access its entries using array indexing notation. "View" means that the slice and the original array point to the same data, i.e, the slice sees changes to the original array and vice versa. "Structured subset" means a cross product of indices along each dimension, as for example a plane or face of a cube. If the original array has dimensions `(N_0, N_1, ..., N_{k-1})`, then a slice views all entries whose indices are `(a_0, a_1, ..., a_{k-1})`, where `a_j` is an ordered subset of `{N_0, N_1, ..., N_j-1}`.
+structured subset of the original array. "Behaves as an array" means that the slice has the same syntax as an array does; one can access its entries using array indexing notation. "View" means that the slice and the original array point to the same data, i.e, the slice sees changes to the original array and vice versa. "Structured subset" means a cross product of indices along each dimension, as for example a plane or face of a cube. If the original array has dimensions $\left(N_0, N_1, ..., N_{k-1}\right)$, then a slice views all entries whose indices are $\left(a_0, a_1, ..., a_{k-1}\right)$, where $a_j$ is an ordered subset of $\left(N_0, N_1, ..., N_j-1\right)$.
```

Comment:
Please confirm you meant to change the curly braces to regular parentheses in the last change on this line

Reply:
no, that was a typo, thanks for catching